### DISCUSSION TITLE
What are the BG color values used for the freeplay menu background

### DISCUSSION THREAD

**Discussion by realvirtu** [FunkinCrew/Funkin#2007] [Q&A]
So ever since Week 7 was released into the public on Newgrounds, I noticed that the color of the background in the Freeplay menu changes to the opponent's color scheme.

What are the hex or rgb values used for the Freeplay background?

I don't know the values, also had a hard time writing this for some reason.

**Comment by Snirozu** [comment]
in Funkin.js there's
``` this.coolColors=[-7179779,-7179779,-14535868,-7072173,-223529,-6237697,-34625,-608764];```
so converted to hex it will be
``` [#9271FD,#9271FD,#223344,#941653,#FC96D7,#A0D1FF,#FF78BF,#F6B604] ```

**Reply by realvirtu** [reply]
Thanks, don't understand how you managed to get them but thanks.
